tumblr/data-lasso
# Update docs and dependencies

# Why make this PR?

This project had a few security vulnerabilities, so I've updated them (primarily Webpack) and made some minor edits to docs.

# How to test

1. `git clone https://github.com/shahkashani/data-lasso`
1. `cd data-lasso`
1. `git checkout shah/update-docs-and-deps`
1. `npm install`
1. `npm run build` - make sure it passes
1. `npm run start`
1. http://localhost:3000
1. Open literally any file and make sure you can use the tool as before, see GIF below for an example:

![lasso](https://user-images.githubusercontent.com/1395893/53183170-d1236580-35c8-11e9-934d-f0fb19f120f0.gif)
